tests/virt_autotest/libvirt_virtual_network_init.pm

```diff
@@ -45,6 +45,12 @@ sub run_test {
         assert_script_run("test $MEM -ge 20", fail_message => "The SUT needs at least 20G FREE MEM for virtual network test");
     }
 
+    #After deployed guest systems, ensure default pool have at least
+    #60Gib available disk space on vm host for virtual network test
+    my $POOL = $self->get_pool_available_space();
+    record_info('Detect Available POOL', $POOL . 'Gib');
+    assert_script_run("test $POOL -ge 60", fail_message => "The SUT needs at least 60Gib available space of default pool for virtual network test");
```

Comment:
suggest "of default pool" -> "of active pool"

Reply:
Great, had updated here already.